src/main/java/net/rptools/maptool/model/library/token/LibraryToken.java

```diff
@@ -305,8 +305,16 @@ public CompletableFuture<Optional<MTScriptMacroInfo>> getMTScriptMacroInfo(Strin
     return new ThreadExecutionHelper<Optional<MTScriptMacroInfo>>()
         .runOnSwingThread(
             () -> {
+              // Cater for the different ways the macro name can be specified
+              String expectedMacroName = macroName;
+              if (macroName.toLowerCase().startsWith("lib://")) {
+                expectedMacroName = macroName.replaceFirst("(?i)lib:\\/\\/.*\\/macro\\/", "");
+              } else if (macroName.toLowerCase().startsWith("lib:")) {
+                expectedMacroName = macroName.substring(4);
+              }
+              ;
```

Comment:
Stray semicolon

Reply:
I called the semicolon catcher, the have rounded it up and removed it